src/app/help/en/welcome.md

```diff
@@ -185,11 +212,11 @@ Your reputation will go up as you earn upvotes for your posts and comments, but
 <a href="#Table_of_Contents">^</a>
 ### <span id="Cashing_out_or_Spending_SBD">Cashing out or Spending SBD</span>
 
-You can spend your SBD at the [Peerhub Store](https://www.peerhub.com/).
+You can exchange SBD for STEEM in the [Currency Market](https://steemit.com/market). Once you have STEEM, you can "Power Up" in your wallet to gain additional Steem Power.
 
-You can exchange your STEEM and SBD for bitcoin on an exchange such as [BlockTrades](https://blocktrades.us/) or [Bittrex](https://bittrex.com/).
+You can spend your SBD at the [Peerhub Store](https://www.peerhub.com/).
 
-You can also "Power Up" and use your STEEM/SBD to gain more Steem Power!
+You can also exchange your STEEM and SBD for bitcoin on an exchange such as [BlockTrades](https://blocktrades.us/) or [Bittrex](https://bittrex.com/).
```

Comment:
These need to be specifically clarified as third party, unaffiliated sites not run by us.

Reply:
(Even if we do this elsewhere in the doc, this portion needs to reiterate it.)